README.md

```diff
@@ -1,3 +1,5 @@
+
+```markdown
```

Comment:
Let me know please, why you add this?

It's broke the markdown:
<img width="1039" alt="image" src="https://github.com/dipiash/eslint-plugin-nimbus-clean/assets/14073668/943283ac-65a2-495e-a0f4-d4ce4e0d58bb">

Reply:
it was a typo sorry!!

Reply:
👍